### ISSUE TITLE
Versioning the webxdc spec and adding a changelog

### ISSUE THREAD

**Issue by hpk42** [webxdc/website#81]
Brought up from @singpolyma and also been under consideration elsewhere. 

It would be useful to start versioning the spec and having a changelog. 

I don't have a ready suggestion but do intend to care for this during july if nobody beats me to it.

**Comment by Simon-Laux** [comment]
The closest thing we currently have for version is the changelog of the typescript types package: [https://github.com/webxdc/webxdc-types/blob/main/CHANGELOG.md](https://github.com/webxdc/website/issues/81#)

If you want some historical stuff instead of just making a version one and only recording future changes. I think it's fine to just start recording future changes as the webxdc community is still small, maybe start before realtime channels, idk.

**Comment by hpk42** [comment]
(nb. i fixed the link in your reply)
The question of how to detect supported features from webxdc apps i consider as a separate issue. 

Versioning of webxdc spec and changelog here is meant to support *implementors* so they get a way to talk and reference it in other docs like "Support webxdc.sendupdate() function as defined in webxdc-spec-1.3". 
And the changelog should be human-readable for implementors and webxdc app developers.

**Comment by trosel** [comment]
Yes, the homepage is really missing a blog or new or changelog. 

There isn’t even a link to the GitHub from the homepage. Think of the effect this has on potential developers. It makes the project look incomplete or dead. 

But that’s not the case, it is actively being developed. Most recent version added real-time p2p something or other. That seems worth a blog post, right?

**Comment by trosel** [comment]
> Once webxdc gets more traction then it makes more sense

I’m not sure how you get more traction without making the project look more alive and worth contributing to. 

I’ve tried to promote this among other decentralized protocols and messengers, but if what they’re seeing is this website, I’m sure they’re not impressed, you know?

Maybe a compromise is just doing the bare minimum to let people know you’re alive. Not necessarily a whole blog with regular pontifications on various topics like roadmap/rationale/etc.
